### PR TITLE
Create/Delete with topo_id as namespace

### PR DESCRIPTION
**Pull request by zzengcs**
Associate each topology deployment request with a unique namespace. Implementing in the function of deploy & delete in Scenario Manager.

Address issue #152

### CODE REVIEW

File: services/merak-topo/handler/handler.go [hunk 1]
```diff
@@ -224,13 +225,19 @@ func Create(k8client *kubernetes.Clientset, topo_id string, aca_num uint32, rack
 		}
 	}
 
-	go Topo_deploy(k8client, aca_image, ovs_image, topo, aca_parameters)
+	namespace = topo_id[:7]
+
```
Comment (cj-chung) [inline services/merak-topo/handler/handler.go:229]: @zzengcs can we use namespace like 'merak-xxxxx` as it's name?

**Review by cj-chung — COMMENTED** [review]
@zzengcs can you test this PR on .111 to make sure entire deployment is working (include vm deployment)? Thanks.

File: services/merak-topo/handler/handler.go (hunk 1 above)
Comment (zzengcs) [reply services/merak-topo/handler/handler.go]: addressed

File: services/merak-topo/handler/handler.go [hunk 2]
```diff
@@ -434,7 +441,8 @@ func Info(k8client *kubernetes.Clientset, topo_id string, returnMessage *pb.Retu
 
 	}
 
-	go UpdateComputenodeInfo(k8client, topo_id)
+	namespace := "merak-" + topo_id[:5]
```
Comment (phudtran) [inline services/merak-topo/handler/handler.go:444]: Please use a constant for this so we can easily change it in the future

File: services/merak-topo/handler/handler.go [hunk 3]
```diff
@@ -509,7 +517,9 @@ func Delete(k8client *kubernetes.Clientset, topo_id string, returnMessage *pb.Re
 
 	}
 
-	go Topo_delete(k8client, topo)
+	namespace := "merak-" + topo_id[:5]
```
Comment (phudtran) [inline services/merak-topo/handler/handler.go:520]: Please use a constant for this so we can easily change it in the future

**Review by phudtran — CHANGES_REQUESTED** [review]
Just one minor change, thanks.

**Comment by zzengcs** [comment]
This PR is solved by PR #173

**Pull request closed without merging**